Makefile

```diff
@@ -45,7 +45,10 @@ KUBE_VERSION ?= 1.21
 KIND_CONFIG ?= kind-$(KUBE_VERSION).yaml
 
 ensure-generate-is-noop: VERSION=$(OPERATOR_VERSION)
-ensure-generate-is-noop: generate bundle
+ensure-generate-is-noop: USER=opentelemetry
+ensure-generate-is-noop: set-image-controller generate bundle
```

Comment:
Added `set-image-controller` target to this check. As it adds the `imageName` to the generated bundle files.  Else the image name will be `controller` by default and the diff check fails.

Reply:
Another option would be to use an env var in the github workflow file, but it wouldn't work for local runs of `make ci`. Your solution is also good, IMO.